scripts/stage-docs.sh

```diff
@@ -172,4 +184,8 @@ gh repo set-default "$(git remote get-url "${REPOSITORY}")"
 echo "Creating pull request to publish release documentation to website."
 # Open PR to merge branch to `main` in `kroxylicious/kroxylicious.github.io`
 BODY="Prepare ${RELEASE_TAG} release documentation for publishing to website"
-gh pr create --head "${RELEASE_DOCS_BRANCH}" --base "${BRANCH_FROM}" --title "Kroxylicious ${RELEASE_TAG} release documentation ${RELEASE_DATE}" --body "${BODY}" --repo "$(gh repo set-default -v)"
+gh pr create --head "${RELEASE_DOCS_BRANCH}" \
+             --base "${BRANCH_FROM}" \
+             --title "Kroxylicious ${RELEASE_TAG} release documentation ${RELEASE_DATE}" \
+             --body "${BODY}" \
+             --repo "$(gh repo set-default -v)"
```

Comment:
I think we should also label the PR with `${RUN_ID_LABEL}` so we can tie all the bits of the run together.

Reply:
I can, but that would mean creating the label in the docs repo too.